test/partitioning.jl

```diff
@@ -390,6 +390,6 @@
 
     d = CartesianGrid{T}(10,10)
     s = slice(d, T(0.5):T(11), T(0.5):T(11))
-    @test s == CartesianGrid(P2(1,1), P2(10,10), dims=(9,9))
+    @test s == CartesianGrid((9,9), P2(0,0), V2(1,1), (0,0))
```

Comment:
Hi @ErickChacon before I start reviewing the PR, could you please explain why some of these old tests are being touched? I had understood that the addition of the new `offset` field should be backward compatible?